src/main/java/com/clubobsidian/dynamicgui/registry/replacer/impl/CooldownReplacerRegistry.java

```diff
@@ -0,0 +1,71 @@
+package com.clubobsidian.dynamicgui.registry.replacer.impl;
+
+import java.util.Collection;
+import java.util.UUID;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.commons.lang3.StringUtils;
+
+import com.clubobsidian.dynamicgui.entity.PlayerWrapper;
+import com.clubobsidian.dynamicgui.manager.dynamicgui.cooldown.Cooldown;
+import com.clubobsidian.dynamicgui.manager.dynamicgui.cooldown.CooldownManager;
+import com.clubobsidian.dynamicgui.registry.replacer.ReplacerRegistry;
+
+public class CooldownReplacerRegistry implements ReplacerRegistry {
+
+	private static CooldownReplacerRegistry instance;
+	
+	public static CooldownReplacerRegistry get()
+	{
+		if(instance == null)
+		{
+			instance = new CooldownReplacerRegistry();
+		}
+		
+		return instance;
+	}
+	
+	private static final String COOLDOWN_PREFIX = "%cooldown_";
+	
+	private Pattern pattern;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/virustotalop/DynamicGui/pullRequest?prid=4627182)